src/hooks/useEdad.js

```diff
@@ -0,0 +1,14 @@
+import {useState} from 'react'
+
+export const useEdad = ( dia, mes, anio) => {
+    const[actual, edad] = useState()
+
+    const CalcularEdad = () => {
+        edad(actual-anio)
```

Comment:
Hello @JessicaPimentelC , hope you are doing well !!  I have a question..   I do not understand where is defined or used this parameter **actual-anio**,  could you explain me a little bit more? or might be the time was not enough?  let me know thankss